pkg/observability/pages/dashboard.vue

```diff
@@ -25,9 +26,13 @@ export default {
     missingCrd() {
       return this.$store.getters['observability/isCrdMissing'];
     },
+    observabilityRepoPresent() {
+      return this.$store.getters['observability/isRepoPresent'];
+    },
   },
   async fetch() {
     await this.$store.dispatch('observability/setMissingCrd', this.missingCrd);
+    await this.$store.dispatch('observability/setRepoPresent', this.observabilityRepoPresent);
```

Comment:
I'm trying to get an idea of that line and it seems like it dispatches an action to set the `repoPresent` value to the current `repoPresent` value, does not it?

Reply:
@hierynomus I believe we do not need the fetch at all, since both setters set values they currently have, those we do not need to show the loading indication as well, here is the commit to drop it all together https://github.com/StackVista/rancher-extension-stackstate/pull/23/commits/386dd979a6981ca9504772d3ffb7f8edb8bdc663

Reply:
I can't remember why I added that in the first place, if it works without 👍🏻